h5p-bildetema-words-grid-view/.storybook/main.js

```diff
@@ -0,0 +1,37 @@
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  stories: ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
+  addons: ["@storybook/addon-links", "@storybook/addon-essentials"],
+  core: {
+    builder: "webpack5",
```

Comment:
Should we use vite for building this one as well? `builder: "@storybook/builder-vite",`

Reply:
sure